src/components/StudyHeader/StudyHeader.jsx

```diff
@@ -3,54 +3,60 @@ import propTypes from 'prop-types';
 import classes from 'classnames';
 import TimeAgo from 'react-timeago';
 import CopyButton from '../CopyButton/CopyButton';
+import {GridContainer} from 'kf-uikit';
 
 const StudyHeader = ({
   kfId,
   modifiedAt,
   shortName,
   name: studyName,
   className,
+  loading,
 }) => {
-  const studyHeaderClasses = classes('StudyHeader', 'BodyContent', className);
+  const studyHeaderClass = classes('bg-lightGrey', className);
+  const topTextClass = classes({
+    'text-lightGrey': loading,
+    topText: !loading,
+  });
   return (
-    <div className="bg-lightGrey">
-      <header className={studyHeaderClasses}>
-        <div>
-          <h3 className="mt-1 text-base">
-            <span className="font-body font-bold pr-4 text-darkGrey" id="kfId">
-              {kfId}
-            </span>
-            <CopyButton text={kfId} />
-            <span className="study-modified-date pl-20 font-light">
-              <small>
-                last updated: <TimeAgo date={modifiedAt} />
-              </small>
-            </span>
-          </h3>
-          <h2 className="mt-4 font-light text-blue text-4xl">
-            {shortName || studyName}
-          </h2>
-        </div>
-        <div>
-          <h4 className="mt-2 text-sm font-bold">Contacts:</h4>
-          <p className="mt-1 font-light">Mary Marazita</p>
-        </div>
-      </header>
+    <div className={studyHeaderClass}>
+      <GridContainer collapsed={true} className="py-8 px-12">
+        <p className={topTextClass}>
+          <b>{kfId}</b>
+          {kfId && <CopyButton text={kfId} className="pl-4 pr-20" />}
+          <small className="font-light">
+            last updated: {modifiedAt && <TimeAgo date={modifiedAt} />}
+          </small>
+        </p>
+        {loading ? (
+          <p className="titleText text-grey">Loading Study ...</p>
+        ) : (
+          <p className="titleText text-blue">{shortName || studyName}</p>
+        )}
+        {!loading && (
+          <div className="row-2 cell-2">
+            <p className="m-0 text-sm font-bold">Contacts:</p>
+            <p className="m-0 font-light">Mary Marazita</p>
+          </div>
+        )}
+      </GridContainer>
     </div>
   );
 };
 
 StudyHeader.propTypes = {
   /** the kf_id (SD_XXXXXXX) for the study  */
-  kfId: propTypes.string.isRequired,
+  kfId: propTypes.string,
```

Comment:
This is for handling the "loading" status. When loading, the study header would show another look for loading, and once data comes in, it would rerender with the data.